File: File-Export-Service/src/main/java/com/kinok0/fileexportservice/repository/UserRepository.java
//package com.kinok0.fileexportservice.repository;
//
//import com.kinok0.fileexportservice.entity.RefreshTokenEntity;
//import com.kinok0.fileexportservice.entity.UserEntity;
//import org.springframework.data.jpa.repository.Query;
//import org.springframework.data.repository.CrudRepository;
//import org.springframework.stereotype.Repository;
//
//import java.util.List;
//import java.util.UUID;
//
//@Repository
//public interface UserRepository extends CrudRepository<UserEntity, UUID> {
//    UserEntity findByLogin(String login);
//
//    @Override
//    List<UserEntity> findAll();
//
//    @Query("select u.login from UserEntity u")
//    List<String> findAllLogins();
//}

File: Profile-Service/src/main/kotlin/com/kinok0/profileservice/repository/UserRepository.kt
package com.kinok0.profileservice.repository

import com.kinok0.profileservice.entity.UserEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface UserRepository : CrudRepository<UserEntity, UUID>{
    fun getUserEntityById(id: UUID) : UserEntity
    fun getUserEntityByLogin(login: String) : UserEntity
}